eslint.config.mjs

```diff
@@ -30,28 +22,33 @@ export default [
                 tsconfigRootDir: './',
             },
         },
-		rules: {
-			// Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
-			// e.g. "@typescript-eslint/explicit-function-return-type": "off",
-			curly: 2,
+        rules: {
+            // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
+            // e.g. "@typescript-eslint/explicit-function-return-type": "off",
+            curly: 2,
 
-			// potentially to fix later see https://trello.com/c/lc8lG7Zj
-			'@typescript-eslint/no-unnecessary-condition': 'off',
-			'@typescript-eslint/no-unsafe-assignment': 'off',
-			'@typescript-eslint/prefer-nullish-coalescing': 'off',
-			'@typescript-eslint/no-unsafe-member-access': 'off',
-			'@typescript-eslint/require-await': 'off',
-			'@typescript-eslint/no-unsafe-enum-comparison': 'off',
-			'@typescript-eslint/no-unsafe-argument': 'off',
-			'@typescript-eslint/no-unsafe-return': 'off',
-			'@typescript-eslint/no-base-to-string': 'off',
-			'@typescript-eslint/prefer-promise-reject-errors': 'off',
-			'@typescript-eslint/no-unsafe-call': 'off',
-			'@typescript-eslint/no-floating-promise': 'off',
-			'@typescript-eslint/restrict-template-expressions': 'off',
-			'@typescript-eslint/no-floating-promises': 'off',
-            '@typescript-eslint/prefer-optional-chain': 'off',
-            '@typescript-eslint/await-thenable': 'off',
-		},
-	},
+			// Since we moved to using the @guardian/eslint-config we are getting many failures on existing code.
+			// These are all rules that we should be using but we are not ready to fix all the existing code yet.
+			// We are using bulk suppressions so we can apply the rules now and fix existing failures later
+            // https://eslint.org/blog/2025/04/introducing-bulk-suppressions/
+			
+            // These are the rules that we may need to fix later see https://trello.com/c/lc8lG7Zj
```

Comment:
can this block be removed now?

Reply:
We could but it is a useful reminder that we still have these items in the block supression document that ought to be fixed over time.

Reply:
Agreed at the SRE meeting to leave in place and added a link to the ESLint Rule docs for ease of decision making.